### ISSUE TITLE
Support for Microsoft Edge/Chromium based and/or Internet Explorer as browser options

### ISSUE THREAD

**Issue by thejims** [willnx/vlab#81]
Could you add support for either IE and/or the Chromium based Microsoft Edge browsers?

I'm able to manually modify the browser in the config.ini as needed for now.
[BROWSER]
agent = chrome
location = C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe

**Comment by willnx** [comment]
Yeah, adding Edge is fine. There's no way in hell I'll add support for IE. Beyond that Microsoft will [stop pushing security updates for it this fall](https://redmondmag.com/articles/2020/08/17/end-of-support-ie11-and-edgehtml.aspx), it's a horrible browser that's caused me too much head ache over the years I worked as a SDE.
